### DISCUSSION TITLE
[0.1300.0.1] Creatures trying to escape into abyss

### DISCUSSION THREAD

**Discussion by RusChosen** [FakeFishGames/Barotrauma#5212] [Unstable feedback]
After the most recent patch (0.1300.0.1) all the creatures that are spawned in the ocean, except for one particular spawn are trying to escape into Abyss right after spawning. They won't bother to attack player's sub and only rarely will target crew, if a crewman happend to be outside at the moment of monsters spawn.
Even a mission target will run down to the bottom and keep ramming it trying to reach for Abyss.
The only exception from my experience is a scripted spawn when the player activates a beacon station. These will properly attack the sub/station, as they should.

I've been playing the game before the most recent patch, and haven't noticed such a problem in 1300.0.0.

**Comment by Phlaxith** [comment]
Same problem, can't really enjoy the game at this time.
It is referring to this issue : https://github.com/Regalis11/Barotrauma/issues/5182

**Comment by CurePindal** [comment]
Can confirm the issue, too. AI isnt engaging and sinking to the sea floor.
